### ISSUE TITLE
Would it be possible to build this utility on Windows for accessibility purposes?

### ISSUE THREAD

**Issue by ScottChesworth** [GoXLR-on-Linux/goxlr-utility#20]
Hi,

I realise the whole drive of this project is to avoid Windows, but bear with me for a second while I explain. :)

I'm researching ways to make GoXLR more usable by some blind streamer friends. These folks use Windows with screen reader software, which speaks in a synthetic voice as they navigate the OS, adjust controls etc. Unfortunately, the official GoXLR app isn't accessible with their screen reader software, and as yet, nobody has managed to get TC Helicon to improve that situation.

So, I'm wondering about your web UI. Browser-based accessibility tends to be one of the more robust areas in terms of support nowadays, I'm thinking that perhaps if I could build this as a portable app that would run in a Windows environment, maybe your web UI could solve this problem?

Any advice appreciated.

**Comment by FrostyCoolSlug** [comment]
I've spent a bit of time looking into this as it seems like an important use-case, unfortunately, due to drivers it's not a simple thing.

If we could find a way to either link the utility directly against the official driver, or prevent the official driver from interacting with the GoXLR while still providing both the audio handling and libusb access it may be possible. But at this point, these aren't things which I have the time to investigate or implement, as they would involve some incredibly complex setups.

So at least for now, and the foreseeable future this isn't something that I'm going to be working on, although that's not to stop someone else picking this up at some point :)

**Comment by ScottChesworth** [comment]
Thanks anyway, really appreciate you taking the time to poke at it. :)

**Comment by FrostyCoolSlug** [comment]
It's been a few months here, but figured I'd give an update if you're still keeping an eye on things.

One of our community members discovered a way to interact with the official GoXLR driver under Windows, and we've since been issuing Windows releases (0.9.0 comes with a Windows installer). In addition, after managing to get it working under Windows, the discussion about accessibility was reopened and I've been working on getting the utilities UI working cleanly with screen readers.

0.9.0 released with limited accessibility (primarily things like loading profiles, routing, volumes and mic settings), but 0.10.0 should be released in the next couple of weeks with a completely accessible UI for all parts of the GoXLR device :)

**Comment by ScottChesworth** [comment]
Yep, still following the repo. I picked up a GoXLR after Piotr tweeted about your progress and I'm using 0.90 here. Have also made sure the streamer friends who I opened this issue for are hooked up too. If there's anything I can do to help around the screen reader UX, just shout. Thanks so much for pressing on with this! :)
